kahuna/public/js/components/gr-panel-button/gr-panel-button-small.html

```diff
@@ -6,7 +6,9 @@
             ng:click="ctrl.hidePanel()"
             ng:class="{'bg-light': !ctrl.state.hidden}"
             gr:tooltip="Hide {{ctrl.name}} panel"
-            gr:tooltip-position="{{ctrl.toolTipPosition()}}">
+            gr:tooltip-position="{{ctrl.toolTipPosition()}}"
+            gr:track-click="{{ctrl.trackingName}}"
+            gr:track-click-data="ctrl.trackingData('Hide panel')">
```

Comment:
I thought so, but as we might have that else where, I didn't want it to clog up the data.